### PR TITLE
fix: Ten/localize list remove

### PR DESCRIPTION
**Pull request by tenbradley**
### Context:
[DE42167](https://rally1.rallydev.com/#/357252966636d/custom/367300408400?detail=%2Fdefect%2F486263074800)
The remove word is not localized in the context menu for list items.

### Quality:
Tested in polarislocalbsi.
- tested English: 'Remove' was shown
- tested French: 'Supprimer' was shown

### CODE REVIEW

**Review by m6jones — APPROVED** [review]
Nice Ten!

**Comment by AKobets** [comment]
Please don't merge, let me test first.

File: components/activity/list/lang/de.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/* eslint quotes: 0 */
+
+export default {
+	"action-remove": "entfernen", // An action to remove a learning task from a list of tasks that are related
```
Comment (AKobets) [inline components/activity/list/lang/de.js:4]: should be capitalized

File: components/activity/list/lang/nl.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+/* eslint quotes: 0 */
+
+export default {
+	"action-remove": "verwijderen", // An action to remove a learning task from a list of tasks that are related
```
Comment (AKobets) [inline components/activity/list/lang/nl.js:4]: should be capitalized

File: components/activity/list/lang/tr.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+/* eslint quotes: 0 */
+
+export default {
+	"action-remove": "soru başlığını kaldır", // An action to remove a learning task from a list of tasks that are related
```
Comment (AKobets) [inline components/activity/list/lang/tr.js:4]: should be capitalized

**Review by AKobets — APPROVED** [review]

**Pull request merged**